.github/workflows/images/price-model.yaml

```diff
@@ -0,0 +1,29 @@
+name: Build and Push Position Manager Docker Image to Docker Hub
+
+on: 
+  push:
+    branches:
+      - main
+    paths:
+      - exec/infrastructure/pricemodel/**
+      - .github/workflows/images/price-model.yaml
```

Comment:
correct, it will not change if we change `pkg/`, as of this PR, the intent is for `pkg/` to be a loosely-coupled, standalone package that we publish and pull in as any other dependency, so that we can pin specifics